components/package.json

```diff
@@ -16,7 +16,11 @@
         "coverage": "yarn test --no-watch --code-coverage",
         "tag:package": "cp ../CHANGELOG.md CHANGELOG.md && npx -p @brightlayer-ui/tag blui-tag -s -blui-angular-components"
     },
-    "peerDependencies": {},
+    "peerDependencies": {
```

Comment:
Looks good! In the future when we migrate this from angular12 to angular13, we will update this section to potentially be a range:

example: `@angular/core:  "^12.0.0" | "^13.0.0"` , something like that.